src/TwoBucks/CalculateGoalsVsPerformance.java

```diff
@@ -1,23 +1,36 @@
-// Two Bucks
-// US10. Calculate Goals vs Actual Performance
-// By Frino Jais
-
+/**
+ * Calculate Goals vs. Actual Performance US.10
+ *
+ * The CalculateGoalsVsPerformance class is used when the user wants
+ * to view their statistics of their performance compared to their
+ * set goals. It will follow the pipe and filter architecture because
+ * it uses data from other portions of the program, and outputs information
+ * based on that data
+ *
+ * @author Frino Jais
+ * @version 1.0
+ * @since 03-04-2020
+ */
 import java.math.RoundingMode;
 import java.text.DecimalFormat;
 import java.util.Scanner;
 import java.util.*;
 
 public class CalculateGoalsVsPerformance {
 
-    private static DecimalFormat df = new DecimalFormat("0.00");
+    private static DecimalFormat df = new DecimalFormat("0.00"); // to keep the values in the correct format
 
-    public void PerformanceAnalysis() {
+    /***
+     * This function will take the data from other classes to show results of how the
+     * user has been spending/saving.
+     */
+    public void PerformanceAnalysis() { // function to show statistics of spending/saving vs actual
 
-        EnterFinancialGoals goals = new EnterFinancialGoals();
-        EnterWeeklyExpense expense = new EnterWeeklyExpense();
-        EnterWeeklyIncome income = new EnterWeeklyIncome();
+        EnterFinancialGoals goals = new EnterFinancialGoals(); // creating an object to retrieve user's goals
+        EnterWeeklyExpense expense = new EnterWeeklyExpense(); //  creating object to retrieve user's expenses
+        EnterWeeklyIncome income = new EnterWeeklyIncome(); // creating object to retrieve user's income
```

Comment:
User's income, goals, and expenses can be retrieved with get methods from users class.

Reply:
Done